src/encoded/auth0.py

```diff
@@ -109,10 +140,9 @@ def login(request):
     else:
         namespace, userid = login.split('.', 1)
 
+    # create new user account if one does not exist
     if namespace != 'auth0':
-        request.session.invalidate()
-        request.response.headerlist.extend(forget(request))
-        raise LoginDenied()
+        userid = signup(request)
```

Comment:
Does this automatically create a new account for you?

Reply:
Yes. If you are validated by an authentication system like google or linkedin but not registered in encoded, an account is created for you.